packages/canvas/canvas-mesh/src/Mesh.ts

```diff
@@ -19,7 +21,15 @@ Mesh.prototype._renderCanvas = function _renderCanvas(renderer: CanvasRenderer):
         this.calculateUvs();
     }
 
-    this.material._renderCanvas(renderer, this);
+    if (this.material._renderCanvas)
+    {
+        this.material._renderCanvas(renderer, this);
+    }
+    else if (!warned)
+    {
+        warned = true;
+        console.warn('Mesh is not supported in canvasRenderer.');
```

Comment:
The wording isn't quite accurate enough, for me.
You see, Mesh is supported... there is a
`Mesh.prototype._renderCanvas`
after all, and on https://pixijs.io/examples/#/mesh-and-shaders/textured-mesh-basic.js you can put { forceCanvas: true } and it works.
So it needs to be a message more about the custom mesh material created not supporting the canvas renderer, rather than just 'mesh' in general